.github/workflows/ci.yaml

```diff
@@ -621,24 +671,32 @@ jobs:
       - name: Run codegen to with_flutter example (non-windows)
         if: ${{ matrix.os.family != 'windows' }}
         working-directory: ./frb_codegen
-        run: cargo run --package flutter_rust_bridge_codegen --bin flutter_rust_bridge_codegen -- --rust-input ../frb_example/with_flutter/rust/src/api.rs --dart-output ../frb_example/with_flutter/lib/bridge_generated.dart --c-output ../frb_example/with_flutter/ios/Runner/bridge_generated.h --dart-format-line-length 120
+        run: cargo run --package flutter_rust_bridge_codegen --bin flutter_rust_bridge_codegen --
+          --rust-input ../frb_example/with_flutter/rust/src/api.rs
+          --dart-output ../frb_example/with_flutter/lib/bridge_generated.dart
+          --c-output ../frb_example/with_flutter/ios/Runner/bridge_generated.h
+          --dart-decl-output ../frb_example/with_flutter/lib/bridge_definitions.dart
+          --dart-format-line-length 120
+          --wasm
         env:
           RUST_LOG: debug
       - name: Run codegen to with_flutter example (windows)
         if: ${{ matrix.os.family == 'windows' }}
         working-directory: ./frb_codegen
-        run: cargo run --package flutter_rust_bridge_codegen --bin flutter_rust_bridge_codegen -- --rust-input ..\\frb_example\\with_flutter\\rust\\src\\api.rs --dart-output ..\\frb_example\\with_flutter\\lib\\bridge_generated.dart --c-output ..\\frb_example\\with_flutter\\ios\\Runner\\bridge_generated.h --dart-format-line-length 120
+        run: cargo run --package flutter_rust_bridge_codegen --bin flutter_rust_bridge_codegen --
+          --rust-input ..\\frb_example\\with_flutter\\rust\\src\\api.rs
+          --dart-output ..\\frb_example\\with_flutter\\lib\\bridge_generated.dart
+          --c-output ..\\frb_example\\with_flutter\\ios\\Runner\\bridge_generated.h
+          --dart-decl-output ..\\frb_example\\with_flutter\\lib\\bridge_definitions.dart
+          --dart-format-line-length 120
+          --wasm
         env:
           RUST_LOG: debug
 
       # #467
-      - name: "Modify cross-platform code (osx) (single block)"
-        if: ${{ matrix.os.family == 'osx' }}
-        # note the "-i '' -e" is a bug of sed specific to MacOS https://stackoverflow.com/questions/19456518
-        run: sed -i '' -e 's/^typedef uintptr_t =.*$/typedef uintptr_t = ffi.UnsignedLong;/g' frb_example/pure_dart/dart/lib/bridge_generated.dart
-      - name: "Modify cross-platform code (non-osx) (single block)"
-        if: ${{ matrix.os.family != 'osx' }}
-        run: sed -i -e 's/^typedef uintptr_t =.*$/typedef uintptr_t = ffi.UnsignedLong;/g' frb_example/pure_dart/dart/lib/bridge_generated.dart
```

Comment:
Btw why do we only need to modify cross-platform for windows, not osx/linux?

Reply:
Something has changed with cbindgen/ffigen that made Windows diverge from other platforms by outputting an `UnsignedLongLong` instead of `UnsignedLong`. I think this might be out of scope, I can revert if needed.

Reply:
I see. Would be great to make it a separate pr so we can track it easily